x/auth/ante/sigverify_test.go

```diff
@@ -29,16 +29,15 @@ import (
 )
 
 func TestSetPubKey(t *testing.T) {
-	suite := SetupTestSuite(t, true)
+	suite := SetupTestSuite(t, false)
```

Comment:
Why are we flipping off `isCheckTx` to false ?

Reply:
Basically the behaviour of `ante.go` in `heimdall v1`  was removed, hence decided to be consistent and set it to `false` everywhere, despite these tests would probably still succeed with `isCheckTx==true`